gammapy/maps/hpx.py

```diff
@@ -875,6 +875,18 @@ def to_cube(self, axes):
         return self.__class__(np.max(self.nside), self.nest, coordsys=self.coordsys,
                               region=self.region, conv=self.conv, axes=axes)
 
+    def pad(self, pad_width):
+        raise NotImplementedError
```

Comment:
Will there ever be pad & crop for the HEALPix classes? Remove these NotImplemented methods?

Reply:
Yes I'm working on them and could potentially add them to this PR.  The logic is somewhat more complex since the "edge" of the map is not as well-defined.